### ISSUE TITLE
Showing result data causes JVM crash

### ISSUE THREAD

**Issue by twothe** [tasgon/observable#12]
Frequently when showing the results of a measurement, the JVM crashes with an OpenGL error (running ATM6 1.7.11 with observable-0.1.3):

```
#  EXCEPTION_ACCESS_VIOLATION (0xc0000005) at pc=0x00007ffbb2de6190, pid=11820, tid=5140

j  org.lwjgl.opengl.GL11C.nglTexImage2D(IIIIIIIIJ)V+0
j  org.lwjgl.opengl.GL11C.glTexImage2D(IIIIIIIILjava/nio/ByteBuffer;)V+17
j  observable.shadow.imgui.impl.gl.ImplGL3.createFontsTexture()Z+121
j  observable.shadow.imgui.impl.gl.ImplGL3.createDeviceObjects()Z+102
j  observable.shadow.imgui.impl.gl.ImplGL3.newFrame()V+11
j  observable.shadow.imgui.impl.gl.ImplBestGL.newFrame()V+4
j  observable.client.ResultsScreen.func_230430_a_(Lcom/mojang/blaze3d/matrix/MatrixStack;IIF)V+22
J 79527 C2 net.minecraft.client.renderer.GameRenderer.func_195458_a(FJZ)V (856 bytes) @ 0x000000000e562fd8 [0x000000000e560f40+0x2098]
J 84680 C2 net.minecraft.client.Minecraft.func_195542_b(Z)V (994 bytes) @ 0x000000000e3cd1bc [0x000000000e3cbd00+0x14bc]
J 76705% C2 net.minecraft.client.Minecraft.func_99999_d()V (208 bytes) @ 0x000000000eb3418c [0x000000000eb340c0+0xcc]
j  net.minecraft.client.main.Main.main([Ljava/lang/String;)V+1392
v  ~StubRoutines::call_stub
j  sun.reflect.NativeMethodAccessorImpl.invoke0(Ljava/lang/reflect/Method;Ljava/lang/Object;[Ljava/lang/Object;)Ljava/lang/Object;+0
j  sun.reflect.NativeMethodAccessorImpl.invoke(Ljava/lang/Object;[Ljava/lang/Object;)Ljava/lang/Object;+100
J 8598 C2 java.lang.reflect.Method.invoke(Ljava/lang/Object;[Ljava/lang/Object;)Ljava/lang/Object; (62 bytes) @ 0x00000000048bd388 [0x00000000048bd2e0+0xa8]
j  net.minecraftforge.fml.loading.FMLClientLaunchProvider.lambda$launchService$0(Lcpw/mods/modlauncher/api/ITransformingClassLoader;[Ljava/lang/String;)Ljava/lang/Void;+50
j  net.minecraftforge.fml.loading.FMLClientLaunchProvider$$Lambda$490.call()Ljava/lang/Object;+12
j  cpw.mods.modlauncher.LaunchServiceHandlerDecorator.launch([Ljava/lang/String;Lcpw/mods/modlauncher/api/ITransformingClassLoader;)V+11
j  cpw.mods.modlauncher.LaunchServiceHandler.launch(Ljava/lang/String;[Ljava/lang/String;Lcpw/mods/modlauncher/TransformingClassLoader;Lcpw/mods/modlauncher/LaunchPluginHandler;)V+57
j  cpw.mods.modlauncher.LaunchServiceHandler.launch(Lcpw/mods/modlauncher/ArgumentHandler;Lcpw/mods/modlauncher/TransformingClassLoader;Lcpw/mods/modlauncher/LaunchPluginHandler;)V+19
j  cpw.mods.modlauncher.Launcher.run([Ljava/lang/String;)V+116
j  cpw.mods.modlauncher.Launcher.main([Ljava/lang/String;)V+37
v  ~StubRoutines::call_stub
```
[crashlog.log](https://github.com/tasgon/observable/files/7070449/crashlog.log)

This crash occurs more frequently the longer Minecraft was already running.

**Comment by tasgon** [comment]
Since this seems like a graphical issue, what OS, graphics card, and driver version are you using?

I'll try and see where this could be caused in my code, but In all likelihood, this is probably an error with the [ImGui library](https://github.com/kotlin-graphics/imgui) I'm using, especially considering this isn't a Java crash but the JVM itself crashing.

**Comment by twothe** [comment]
Windows 10 on NVidia Geforce 1070 version 466.27.

I doubt it is linked to this driver though.

**Comment by tasgon** [comment]
> On a side-note: I wonder why you don't just open a standard Java Window?

What, do you mean opening a new window entirely for the results?

**Comment by tasgon** [comment]
I haven't seen or heard anyone else with the same issue, so I'm going to close this for now. If it happens again, reopen this issue to let me know.

**Comment by bigenergy** [comment]
I think I have the same problem
https://gist.github.com/bigenergy/1c8c088d0587e260a725031a663dca74

**Comment by bigenergy** [comment]
@tasgon

**Comment by ghost** [comment]
I'm having the same issue 
https://gist.github.com/fabmm/b2c034cd7edac2cea769d2d300dfd0ed

**Comment by tasgon** [comment]
This is fixed in 1.19 now, and should be backported to 1.18 relatively soon™️. Not sure if this will land for 1.16 as well, though, but if I can scrounge up the time, I'll give it a try.
